src/testoptions.c

```diff
@@ -395,6 +395,9 @@ int initialize_tests(Connection *ctl, TestOptions *options, char *buff,
   if (useropt & TEST_META) {
     add_test_to_suite(&first, buff, buff_strlen, TEST_META);
   }
+  // Meta data that should be unconditionally saved is set here.
```

Comment:
The other has to do with the format used to request tests from the server, but has no bearing on any aspect of the tests performed. (Do we speak the raw protocol format or the JSON format?). In the spirit of full disclosure, I could absolutely add that, but it should not be a blocker for this release.